corehq/apps/domain/forms.py

```diff
@@ -822,82 +822,82 @@ def save(self, domain):
 
 
 class DomainInternalForm(forms.Form, SubAreaMixin):
-    sf_contract_id = CharField(label=ugettext_noop("Salesforce Contract ID"), required=False)
-    sf_account_id = CharField(label=ugettext_noop("Salesforce Account ID"), required=False)
-    initiative = forms.MultipleChoiceField(label=ugettext_noop("Initiative"),
```

Comment:
I was adding a bunch of strings and really didn't want to translate those, but it felt weird to leave these strings translated.  This is a form that's only accessible by Dimagi employees and is meant to be filled out in English.  Translations are done either by Dimagi staff or by a paid external translator who charges by the word.  Any change to a string requires a retranslation, and the translators don't have a way of knowing the urgency of translating a string.In short, a human has to do work for every string that's marked for translations, and for internal pages, we don't get any benefit from that work.